src/intTest/resources/edifact/multi_pathology.edifact.dat

```diff
@@ -1,4 +1,4 @@
-UNB+UNOC:3+XX11+000000024600002:80+100301:1751+1015++MEDRPT++1'
+UNB+UNOC:3+000000004400001+000000024600002:80+100301:1751+1015++MEDRPT++1'
```

Comment:
Here and in other examples:

Do we need the :80 after the 15 digit codes? The edifact word document seems to suggest so

Reply:
Yes we do, but we'll need to make a change to the interchange header parser (i think). Outside of this PR, i still have a couple of things to do before this ticket is complete,  so I'll create a separate PR for the `:80` part.